src/_gettsim/interface.py

```diff
@@ -373,36 +373,36 @@ def _add_rounding_to_function(
     """
     func = copy.deepcopy(input_function)
     nice_name = ".".join(path)
-    leaf_name = path[-1]
+    qualified_name = "__".join(path)
```

Comment:
Don't make a habit out of this (=use stuff from dags/for now flatten dict), but for this quick'n'dirty one out is perfectly fine!